apps/frontend/src/libs/components/app/app.tsx

```diff
@@ -19,11 +21,14 @@ import {
 } from "~/libs/hooks/hooks.js";
 import { actions as userActions } from "~/modules/users/users.js";
 
+import { Modal } from "../modal/modal.js";
+
 const App = (): JSX.Element => {
 	const dispatch = useAppDispatch();
 	const { pathname } = useLocation();
 	const dataStatus = useAppSelector(({ users }) => users.dataStatus);
 	const users = useAppSelector(({ users }) => users.users);
+	const [isModalOpen, setIsModalOpen] = useState<boolean>(false);
```

Comment:
I think, it's better to keep track this state inside `Modal` component itself, and maybe create custom hook `useModal` which will return essential data for keep tracking and controlling modal. For example
`const [isOpened, showModal] = useModal()`
And `isOpened` then will be used to determine whether or not to render `Modal` component, and `showModal` can be passed to some `Button` component as `onClick` prop, so when you press the button modal become opened and renders